easybuild/easyconfigs/a/ABAQUS/ABAQUS-2024-hotfix-2405.eb

```diff
@@ -0,0 +1,44 @@
+name = 'ABAQUS'
+version = '2024'
+local_hotfix = '2405'
+versionsuffix = '-hotfix-%s' % local_hotfix
+
+homepage = 'https://www.simulia.com/products/abaqus_fea.html'
+description = """Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit
+ dynamics FEA."""
+
+toolchain = SYSTEM
+
+sources = [
+    '%(version)s.AM_SIM_Abaqus_Extend.AllOS.1-6.tar',
+    '%(version)s.AM_SIM_Abaqus_Extend.AllOS.2-6.tar',
+    '%(version)s.AM_SIM_Abaqus_Extend.AllOS.3-6.tar',
+    '%(version)s.AM_SIM_Abaqus_Extend.AllOS.4-6.tar',
+    '%(version)s.AM_SIM_Abaqus_Extend.AllOS.5-6.tar',
+    '%(version)s.AM_SIM_Abaqus_Extend.AllOS.6-6.tar',
+    # hotfixes
+    '%%(version)s.FP.CFA.%s.Part_SIMULIA_EstPrd.Linux64.tar' % local_hotfix,
+    '%%(version)s.FP.CFA.%s.Part_SIMULIA_FlexNet.Linux64.tar' % local_hotfix,
+    '%%(version)s.FP.CFA.%s.Part_SIMULIA_Readme.AllOS.tar' % local_hotfix
+]
+checksums = [
+    {'2024.AM_SIM_Abaqus_Extend.AllOS.1-6.tar': 'a8fcd10541a90177aefe68f0dee2a675a56cb97e1fdf4fb7d864b41d594f8b19'},
+    {'2024.AM_SIM_Abaqus_Extend.AllOS.2-6.tar': '6caffc60ee34351203ac4205b4fcdc7e9975842e35d3ce689b831f94929ddac4'},
+    {'2024.AM_SIM_Abaqus_Extend.AllOS.3-6.tar': '3d3bc97c686af0c87c4b6e46ae330983ed515bcc180dd30a834811c458774347'},
+    {'2024.AM_SIM_Abaqus_Extend.AllOS.4-6.tar': 'c62596bd99125475e97bdeccd71baaf7cb36322e8da4d0dd1e4029bac677be16'},
+    {'2024.AM_SIM_Abaqus_Extend.AllOS.5-6.tar': 'ee3349b2407b5d8a315108656590f39890d82c47085235b847103e22fba96fea'},
+    {'2024.AM_SIM_Abaqus_Extend.AllOS.6-6.tar': '482829b1c364966ae01a577233bddf6b377b8e3cd9b7f77dd95830d4636677d7'},
+    {'2024.FP.CFA.2405.Part_SIMULIA_EstPrd.Linux64.tar':
+     '70231acf27506546174b896b6724f27a14673594b791eb62fb4c1e3b24ba1852'},
+    {'2024.FP.CFA.2405.Part_SIMULIA_FlexNet.Linux64.tar':
+     '7295d883071bc777e871484e4471e36b6b588bebbe30bb65a0954bdbb676b5cb'},
+    {'2024.FP.CFA.2405.Part_SIMULIA_Readme.AllOS.tar':
+     '3e9009db5423bb7df4db8080e2cb717a1807e7be18125db2d9c78a95f7e3434c'},
+]
+
+download_instructions = f"""{name} requires manual download from https://www.3ds.com/support/software-downloads.
```

Comment:
Unfortunately we'll need to wait with this approach until we merged the 5.0 branch and fully dropped python2 support.